taprpc/mintrpc/mint.proto

```diff
@@ -102,11 +102,17 @@ message MintingBatch {
     */
     bytes batch_key = 1;
 
-    // The assets that are part of the batch.
-    repeated MintAsset assets = 2;
+    /*
+    The transaction ID of the batch. Only populated if the batch has been
+    committed.
+    */
+    string batch_txid = 2;
```

Comment:
Do we care about breaking gRPC backward compatibility at this point or not yet?

Reply:
I think not necessarily yet? Just felt aesthetically better to group the singular fields, then have the repeated fields at the end.